cmd/containerboot/main.go

```diff
@@ -1287,6 +1321,19 @@ func isOneStepConfig(cfg *settings) bool {
 	return cfg.TailscaledConfigFilePath != ""
 }
 
+// isTCPProxy returns true if the Tailscale node needs to be configured to act
+// as a TCP proxy, proxying to an endpoint provided via one of the config env
+// vars.
+func isTCPProxy(cfg *settings) bool {
```

Comment:
Maybe `isL3Proxy` to save confusion about UDP?

Reply:
Yeah that would be more accurate thanks 👍🏼
(I think I started using 'TCP proxy' wording in places because we had been thinking about replacing the iptables/nftables rules with a proxy implemented on top of tailscale serve and there already is a [`tcpHandlerForServe`](https://github.com/tailscale/tailscale/blob/v1.69.0-pre/ipn/ipnlocal/serve.go#L420) that does some of what we need)